youtube_dl/extractor/googledrive.py

```diff
@@ -271,6 +287,9 @@ def add_source_format(urlh):
             'title': title,
             'thumbnail': 'https://drive.google.com/thumbnail?id=' + video_id,
             'duration': int_or_none(get_value('length_seconds')),
+            'created_date': createdTime,
+            'modified_date': modifiedTime,
+            'owners': owner_lst,
```

Comment:
`owners` is not a recognised yt-dl metadata item. What is its significance?

Consider using `uploader` or `creator` instead (with value `', '.join(owner_lst or [])`).